orcestra_book/science.md

```diff
@@ -4,27 +4,120 @@ orphan: true
 
 # Science
 
-## The ITCZ
-ITCZ is the rising branch of the Hadley circulation, marked by low-level convergence, deep convection, and precipitation.
-Its importance for the climate system can hardly be exaggerated.
-The ITCZ plays a central role in the global circulation system and around 1/3 of the global precipitation falls within the latitudinal band associated with the ITCZ.
-While the ITCZ is expected to change in a warming climate, our confidence in the ability of climate model to correctly represent this trend is limited as climate models have struggled for decades to correctly represent the ITCZ.
-Due to this central role, the ITCZ has, of course, already been studied intensely and the classic picture that arises is that of a continuous band of precipitating clouds spanning from east to west. However, the figure below shows the day-to-day variability of the ITCZ revealing a more chaotic picture with significant changes in cloud amount and organisation from one day to the next.
-
-```{figure} /figures/itcz_daily_variability.png
-:alt: The day-to-day variability of the ITCZ
-:width: 600px
-:align: center
-
-Snapshots from VIIRS (NOAA-20) from the 1st of July 2021 until the 15th of July 2021 in the East Atlantic (26°W to 20°W, 0° to 14°N).
+## The scientific motivation for PERCUSION
+![](./figures/ITCZ_ferrel.jpg)
+*Figure 1. William Ferrel's schematic depiction of Earth’s surface wind. Figure taken from Birner et al, 2014 and adapted from Ferrel, 1856.*
+
+Most tropical rain is concentrated within a narrow latitudinal band known as the Inter-Tropical Convergence Zone (ITCZ). This band shifts in position, width and longitude on timescales from days to millenia, with consequences for the billions of people who live within its reach.
+
+These variations in the ITCZ have been linked to changes in the atmosphere's energy balance, but this balance is itself hard to constrain --- it depends on mesoscale circulations which are notoriously difficult to measure (vertical winds are usually an order of magnitude smaller than horizontal winds). Without these observations, we are unable to constrain our theories and computer simulations, limiting our ability to predict future changes in the width and position of the ITCZ.
+
+Will the islands of Cape Verde, which lie at the edge of the ITCZ, get wetter or dryer in the next 50 years? What about about the Sahel? We cannot yet confidently say what will happen.
+
+To break this deadlock, one of PERCUSION's scientific objectives is to measure the mesoscale vertical velocity $w$ at the center and edges of the ITCZ. Variability in $w$ will be characterized over 20+ flights between August and October in the East and West Atlantic. This will allow, for the first time, a process-level observational estimate of the ITCZ's circulation and energy export. With this information, we will be able to peer into the inner life of the ITCZ and understand why it billows and blows.
```

Comment:
```suggestion
To break this deadlock, one of PERCUSION's scientific objectives is to measure the mesoscale vertical velocity $w$ at the center and edges of the ITCZ. Variability in $w$ will be characterized over 20+ flights in August and September in the East and West Atlantic. This will allow, for the first time, a process-level observational estimate of the ITCZ's circulation and energy export. With this information, we will be able to peer into the inner life of the ITCZ and understand why it billows and blows.
```